tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateDateTimeTestData.java

```diff
@@ -0,0 +1,104 @@
+          emptyMapStringToObject.plus("dateStyle", "long"),
+          emptyMapStringToObject.plus("timeStyle", "long"),
+          emptyMapStringToObject.plus("hour", "numeric"),
+          emptyMapStringToObject.plus("hour", "numeric").plus("minute", "numeric"),
+          emptyMapStringToObject.plus("hour", "numeric").plus("minute", "numeric")
+              .plus("second", "numeric"),
+          emptyMapStringToObject.plus("hour", "numeric").plus("minute", "numeric")
+              .plus("second", "numeric").plus("fractionalSecondDigits", 1),
+          emptyMapStringToObject.plus("hour", "numeric").plus("minute", "numeric")
+              .plus("second", "numeric").plus("fractionalSecondDigits", 2),
+          emptyMapStringToObject.plus("hour", "numeric").plus("minute", "numeric")
+              .plus("second", "numeric").plus("fractionalSecondDigits", 3),
+          emptyMapStringToObject.plus("hour", "numeric").plus("minute", "numeric")
+              .plus("second", "numeric"),
+          emptyMapStringToObject.plus("month", "numeric").plus("weekday", "long")
+              .plus("day", "numeric"),
+          emptyMapStringToObject.plus("month", "2-digit").plus("weekday", "long")
+              .plus("day", "numeric"),
+          emptyMapStringToObject.plus("month", "long").plus("weekday", "long").plus("day", "numeric"),
+          emptyMapStringToObject.plus("month", "short").plus("weekday", "long")
+              .plus("day", "numeric"),
+          emptyMapStringToObject.plus("month", "narrow").plus("weekday", "long")
+              .plus("day", "numeric"),
+          emptyMapStringToObject.plus("month", "short").plus("weekday", "short")
+              .plus("day", "numeric"),
+          emptyMapStringToObject.plus("month", "short").plus("weekday", "narrow")
+              .plus("day", "numeric"),
+          emptyMapStringToObject.plus("era", "long"),
+          emptyMapStringToObject.plus("era", "short"),
+          emptyMapStringToObject.plus("era", "narrow"),
+          emptyMapStringToObject.plus("timeZoneName", "long"),
+          emptyMapStringToObject.plus("timeZoneName", "short"),
+          emptyMapStringToObject.plus("timeZoneName", "shortOffset"),
+          emptyMapStringToObject.plus("timeZoneName", "longOffset"),
+          emptyMapStringToObject.plus("timeZoneName", "shortGeneric"),
+          emptyMapStringToObject.plus("timeZoneName", "longGeneric")
+      ));
+
+  private static final PVector<String> CALENDARS =
+      TreePVector.from(Arrays.asList(
+          "gregory",
+          "buddhist", "hebrew", "chinese", "roc", "japanese",
+          "islamic", "islamic-umalqura", "persian"
+      ));
+
+  private static final PVector<String> TIME_ZONES =
+      TreePVector.from(Arrays.asList(
+          "America/Los_Angeles",
+          "Africa/Luanda",
+          "Asia/Tehran",
+          "Europe/Kiev",
+          "Australia/Brisbane",
+          "Pacific/Palau",
+          "America/Montevideo"
+      ));
```

Comment:
other code uses ImmutableSet.of("…", "…", "…") - does this provide value beyond that?

Reply:
I switched over to using Guava according to Mark's guidance, so it's a moot point now.

(There's a bit of cloning during runtime and verbosity of code that could be avoided if one were to use persistent data structures.)